testsuite/cli/src/client_proxy.rs

```diff
@@ -182,11 +182,12 @@ impl ClientProxy {
             Some(dd_account_data)
         };
 
-        let faucet_url = Url::parse(&faucet_url.unwrap_or(format!(
-            "http://{}",
-            url.host_str().unwrap().replace("client", "faucet").as_str()
-        )))
-        .expect("Invalid faucet URL");
+        let faucet_url = if let Some(faucet_url) = &faucet_url {
+            Url::parse(faucet_url).expect("Invalid faucet URL specified")
+        } else {
+            url.join("/mint")
```

Comment:
this will end with /v1/mint right?

Reply:
It's a URL join, so because it starts with `/` it sets the absolute path. "https://testnet.libra.org/v1" will become "https://testnet.libra.org/mint".